bucket/rapidcrc.json

```diff
@@ -0,0 +1,39 @@
+{
+  "homepage": "https://ov2.eu/programs/rapidcrc-unicode",
+  "description": "An open source CRC/MD5/SHA hashing program",
+  "license": {
+    "identifier": "GPL-2.0-only",
+    "url": "https://github.com/OV2/RapidCRC-Unicode/blob/master/LICENSE"
+  },
+  "version": "0.3.33",
+  "architecture": {
+    "64bit": {
+      "url": "http://dl.ov2.eu/programs/rapidcrc-unicode/RapidCRC-Setup-0.3.33-x64.msi",
+      "hash": "f2a357c815a2c300f11b5899d310178e28a74a79be13cb7e7e2b713789f32094"
+    },
+    "32bit": {
+      "url": "http://dl.ov2.eu/programs/rapidcrc-unicode/RapidCRC-Setup-0.3.33.msi",
+      "hash": "83367527bafae2def12c08cf9fcec29a16a94912d5c9f096cea89e3b5f2a82fd"
+    }
+  },
+  "extract_dir": "PFiles/RapidCRC",
+  "shortcuts": [
+    [
+      "RapidCRC.exe",
+      "RapidCRC"
+    ]
+  ],
+  "checkver": {
+    "github": "https://github.com/OV2/RapidCRC-Unicode"
+  },
+  "autoupdate": {
+    "architecture": {
+      "64bit": {
+        "url": "http://dl.ov2.eu/programs/rapidcrc-unicode/RapidCRC-Setup-$version-x64.msi"
+      },
+      "32bit": {
+        "url": "http://dl.ov2.eu/programs/rapidcrc-unicode/RapidCRC-Setup-$version.msi"
+      }
+    }
+  }
```

Comment:
```suggestion
    "version": "0.3.33",
    "description": "Hash calculator with unicode support",
    "homepage": "https://ov2.eu/programs/rapidcrc-unicode",
    "license": "GPL-2.0-only",
    "architecture": {
        "64bit": {
            "url": "http://dl.ov2.eu/programs/rapidcrc-unicode/rapidcrc_0.3.33_x64.rar",
            "hash": "9f555b350eee87555a751fa6408f182b89a052771d8ab7c9e5f8fd13508f0ed9"
        },
        "32bit": {
            "url": "http://dl.ov2.eu/programs/rapidcrc-unicode/rapidcrc_0.3.33.rar",
            "hash": "8401f0f9e2501e3a1d93b6903e8e3539834752e201594b30befe355a9c376a7a"
        }
    },
    "shortcuts": [
        [
            "RapidCRC.exe",
            "RapidCRC"
        ]
    ],
    "checkver": ">v([\\d.]+)</",
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "http://dl.ov2.eu/programs/rapidcrc-unicode/rapidcrc_$version_x64.rar"
            },
            "32bit": {
                "url": "http://dl.ov2.eu/programs/rapidcrc-unicode/rapidcrc_$version.rar"
            }
        }
    }
```

Reply:
Thanks, `.rar` wasn't listed on the wiki, so I thought it's unsupported.